docs/docusaurus.config.js

```diff
@@ -123,6 +123,10 @@ const config = {
                 label: "Installation",
                 to: "/docs/installation",
               },
+              {
+                label: "API Reference",
+                to: "/docs/api-reference",
```

Comment:
```suggestion
                to: "/api-reference-docs/",
```
This would make it link to the location it's currently building to, if that's where we want to keep it.